### PR TITLE
Feature/BOAS 313 case details project information

### PR DESCRIPTION
**Pull request by DavidJWhiteDev**
## Describe your changes

Case Details - Project Information page 

## BOAS-313 Case Details - Project Information
https://pins-ds.atlassian.net/browse/BOAS-313

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/case/project-information.njk [hunk 1]
```diff
@@ -1,11 +1,170 @@
...
+	}) }}
 
-	<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 {% endblock %}
+
+{% macro writeChangeLink(params) %}
+	{% set userIsInspector = domainType === 'inspector' %}
+
+	{% if not userIsInspector %}
+		{% set baseUrl = '/applications-service/case/' + applicationId +'/edit/' %}
```
Comment (lutamKC) [inline apps/web/src/server/views/applications/case/project-information.njk:149]: please use the url filter.
should be `'applications-edit|url({applicationId, step})`

Comment (DavidJWhiteDev) [reply apps/web/src/server/views/applications/case/project-information.njk:149]: @lutamKC yes! I was just thinking that, and then I saw this comment!  Will do!

**Review by lutamKC — APPROVED** [review]

**Pull request merged**